### PR TITLE
tf-summarize: init at 0.3.2

### PR DESCRIPTION
**Pull request by pjrm**
###### Description of changes

new package: tf-summarize (https://github.com/dineshba/tf-summarize)

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/cluster/tf-summarize/default.nix
Comment (benley) [inline pkgs/applications/networking/cluster/tf-summarize/default.nix:25]: does `tf-summarize -v` print "development" instead of its actual package version?  If so that may suggest that your ldFlags passing main.version isn't having the intended effect

Comment (pjrm) [reply pkgs/applications/networking/cluster/tf-summarize/default.nix:25]: Yes, we can see it here: https://github.com/dineshba/tf-summarize/blob/v0.3.2/main.go#L14

That makes sense, and I will update that.

File: pkgs/applications/networking/cluster/tf-summarize/default.nix
Comment (benley) [inline pkgs/applications/networking/cluster/tf-summarize/default.nix:32]:
```suggestion
    maintainers = with maintainers; [ pjrm ];
```
This has to match the attribute you set in maintainer-list.nix

**Review by benley — APPROVED** [review]

**Comment by benley** [comment]
crap, I forgot to fix the commit title to "init _at_ 0.3.2" instead of "init a 0.3.2" before clicking merge.  Accidentally clicked too quickly.

**Comment by benley** [comment]
Thank you for your first nixpkgs PR, @pjrm :-)

**Comment by pjrm** [comment]
Thank you very much for your help @benley :D

**Pull request merged**
